packages/core/core/src/utils/transform-content-types-to-models.ts

```diff
@@ -246,20 +287,20 @@ export const transformContentTypesToModels = (contentTypes: LoadedContentTypeMod
     });
 
     if (hasComponentsOrDz(contentType)) {
-      const compoLinkModel = createCompoLinkModel(contentType);
+      const compoLinkModel = createCompoLinkModel(contentType, options);
       models.push(compoLinkModel);
     }
 
     const model: Model = {
       uid: contentType.uid,
       singularName: contentType.modelName,
-      tableName: identifiers.getTableName(contentType.collectionName),
+      tableName: contentType.collectionName, // This shortening will be handled in metadata
```

Comment:
So the model still has the long name, but the db metadata has a short one? Will the original model be modified or not?

Reply:
Oooh, nice catch, this comment was a lie / left over from when metadata did shorten the table names. But that has changed and now this has to be shortened. I'm fixing this and will try to come up with some better tests too.

Reply:
Update: no, the code was correct, the comment was still misleading. it's handled in loadModels() not createMetadata(), so I will specify that so it's clear